ci/docker-run.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+ # This is intended to be run after docker-setup.sh has been used to setup the docker container(s)
+set -ex
+
+# TEST_MODE should be one of "unit" or "integration" (defaults to "unit" unless INTEGRATION=true)
+: "${TEST_MODE:=$([[ "${INTEGRATION}" = "true" ]] && echo "integration" || echo "unit")}"
+export TEST_MODE
+
+docker-compose -f "ci/common/docker-compose.yml" -f "ci/${TEST_MODE}/docker-compose.override.yml" up --exit-code-from logstash
```

Comment:
nice, for .ci we might want to try generalizing the idea even a bit further.
would be nice if there was a way to get docker to print the final configuration to console.

Reply:
If we add `--verbose` we can at least get a log line indicating which files it is attempting:

~~~
compose.config.config.find: Using configuration files: ./ci/common/docker-compose.yml,./ci/unit/docker-compose.override.yml
~~~